File: Code/Задание 7.py
s = float(input())
k = int(input())
if k == 1:
    c = 90
elif k == 2:
    c = 96
elif k == 3:
    c = 112
else:
    print('Неверный код валюты')
    exit()
res = s / c
print(f'{res:.2f}')

File: Code/Задание 7.pas
var
  s, res: real;
  k: integer;
  c: real;
begin
  readln(s, k);
  case k of
    1: c := 90;
    2: c := 96;
    3: c := 112;
  else
    writeln('Неверный код валюты');
    exit;
  end;
  res := s / c;
  writeln(res:0:2);
end.
